samples/basic/blink_led/README.rst

```diff
@@ -8,10 +8,10 @@ Overview
 
 This is a sample app which blinks a LED using PWM.
 
-The LED will start at a blinking frequency of 0.5Hz. Every 4 seconds,
+The LED will start at a blinking frequency of 1 Hz. Every 4 seconds,
 the blinking frequency will double. When the blinking frequency
-reaches 500Hz, the blinking frequency will be halved every 4 seconds
-until the blinking frequency reaches 0.5Hz. This completes a whole
+reaches 64 Hz, the blinking frequency will be halved every 4 seconds
+until the blinking frequency reaches 1 Hz. This completes a whole
```

Comment:
can you tweak the last sentence to read:

     This faster-then-slower LED blinking cycle repeats forever.